README.md

```diff
@@ -1,133 +1,90 @@
-## Step by Step Guide to make your own Gemini API Discord Bot
+<p align="center"><img height="160" src="assets/logo.png" /></p>
 
-Try the Gemini Bot by adding to your discord server : [Click here to Add](https://discord.com/oauth2/authorize?client_id=1214629041135419402&permissions=36507479040&scope=bot)
+## Discord Gemini Bot 🤖
 
-### To Use the Bot
-After adding the bot to your discord server then you can simply tag the bot and ask question as below after which you will get your answers shortly 😉
-![alt text](Images/image-3.png)
+### Features
 
-The Bot won't give any response as currently it only works locally. Will host soon at Replit:D
+<p align="center"><img width="600" src="assets/screenshot_1.png" /></p>
 
-### Setup Initial Coding Workspace
-Execute this commands on your IDE on a directory you want to work on
-1. ```npm init```
-2. ```npm install discord.js```
-3. ```npm install @google/generative-ai```
+- [x] 😀 Chat using texts
+- [x] 👋🏼 Chat with context
+- [x] 🥷🏼 Clear context with `/clear` command
+- [x] 🚀 Streaming output
+- [ ] Multimode support
 
-After doing this:
+<br />
 
-### Making APIs Ready
+### Deploy
 
-4. Get Discord API Key via : https://discord.com/developers/applications
+#### Get the codes
 
-After you create a API key then you will get to see a token once so you should copy that and keep it safely somewhere, better to use Google Keep - https://keep.google.com/ for this.
+This project is writeen in TypeScript, before deploy, ensure you have NodeJS (>= v20 is recomment) installed. If you don't have NodeJS, [fnm](https://github.com/Schniz/fnm) is the NodeJS installer recommended.
```

Comment:
typo: is **written**; is **recommended**

Reply:
done